widget/embedded/src/components/Quote/Quote.tsx

```diff
@@ -300,13 +306,19 @@ export function Quote(props: QuoteProps) {
                   steps[steps.length - 1].to.token.displayName
                 }`}
               </Typography>
-              <Typography
-                color="$neutral600"
-                ml={2}
-                size="xsmall"
-                variant="body">
-                {`($${roundedOutputUsdValue})`}
-              </Typography>
+              <Tooltip
+                content={formatTooltipNumbers(output.usdValue || '')}
+                style={{
+                  display: 'flex',
+                }}>
+                <Typography
+                  color="$neutral600"
+                  ml={2}
+                  size="xsmall"
+                  variant="body">
+                  {`($${roundedOutputUsdValue})`}
+                </Typography>
+              </Tooltip>
```

Comment:
It would be advisable to pass the swap-box as the tooltip container, similar to other instances. If the Tooltip component is used without the container prop being passed, the tooltip will be appended to the body, and themes won't be applicable to the tooltip.
you can use getContainer helper.
![image](https://github.com/rango-exchange/rango-client/assets/120931880/b2a64d39-40c6-431f-bd38-672a14e4f425)

Reply:
I believe you can achieve the same result by overriding the styles of the typography component, which would help in keeping the tooltip props simpler.

Reply:
fixed.